### PR TITLE
Changes for the 4.4.3 release

### PR DESCRIPTION
**Pull request by dmsurti**
1. Bump up the version number.
2. Update the change log.

### CODE REVIEW

File: docs/CHANGES.txt [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+
+22 Jul 2015 (IT)
+    - Update the docs with known bugs, issues list and describe the PR
+      workflow on github for development and documentation.
+
+20 Jul 2015 (DS)
+    - Fix TVTK wrapper to update number of cells when a cell is inserted in a
+      VTK cell array
+
+20 Jul 2015 (IT)
```
Comment (itziakos) [inline docs/CHANGES.txt]: This change is obsolete since we now use wx-tools in travis-ci, I do not think we need to add this update.

**Comment by itziakos** [comment]
On small comment apart from that :+1:

**Comment by prabhuramachandran** [comment]
Yes, agree with @itziakos , otherwise LGTM.

**Comment by prabhuramachandran** [comment]
:+1:

**Pull request merged**